scheduler/feasible.go

```diff
@@ -198,6 +199,10 @@ func NewCSIVolumeChecker(ctx Context) *CSIVolumeChecker {
 	}
 }
 
+func (c *CSIVolumeChecker) SetJob(job *structs.Job) {
```

Comment:
Isn't it `SetNamespace` (if we're comparing with `SetVolumes` below)? Or maybe `SetNamespaceFromJob` if you want to get really wordy. 😀

Reply:
Well, this way it implements `ContextualIterator`, which is what I was going for. Looking again, it doesn't seem to matter.